testing/test-scripts/helloworld-v1-v2-route-rules.yaml

```diff
@@ -24,3 +24,5 @@ spec:
       - weight: 0.25
         tags:
         - "version=v2"
+      - tags:
+        - "version=v1"
```

Comment:
The script clears out the previous rules before the next tests.  Without setting version=v1 in that case 3/4 of the requests get a 404.  I can update the script to keep the default routing rule and then remove that.